CHANGELOG.md

```diff
@@ -2,155 +2,188 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
Should I delete this too?

Reply:
I think we still aim to follow Semantic Versioning -- perhaps I am out of the loop? The fact it wasn't always followed (especially on the way to v1.0.0!) I don't see as changing that?